code/modules/admin/topic.dm

```diff
@@ -818,7 +818,12 @@
 		else
 			dat += "<td width='20%'><a href='?src=[REF(src)];[HrefToken()];jobban3=posibrain;jobban4=[REF(M)]'>Posibrain</a></td>"
 
-
+		//Sentience Potion Spawn
+		if(jobban_isbanned(M, ROLE_SENTIENCE))
+			dat += "<td width='20%'><a href='?src=[REF(src)];[HrefToken()];jobban3=[ROLE_SENTIENCE];jobban4=[REF(M)]'><font color=red>[ROLE_SENTIENCE]</font></a></td>"
+		else
+			dat += "<td width='20%'><a href='?src=[REF(src)];[HrefToken()];jobban3=[ROLE_SENTIENCE];jobban4=[REF(M)]'>[ROLE_SENTIENCE]</a></td>"
+		
 		//Deathsquad
 		if(jobban_isbanned(M, "deathsquad"))

```

Comment:
convert this into a define while you're here

Reply:
oh and the similar blocks for lavaland, drone, posibrain too. I can't mark them because they don't show in the diff.